data/virt_autotest/guest_unattended_installation_files/sles_12_plus_kvm_hvm_guest_graphical_x86_64.xml

```diff
@@ -126,42 +126,54 @@
     <services>
       <disable config:type="list"/>
       <enable config:type="list">
+        <service>cron</service>
+        <service>display-manager</service>
+        <service>nscd</service>
+        <service>postfix</service>
+        <service>purge-kernels</service>
+        <service>rollback</service>
+        <service>rsyslog</service>
         <service>sshd</service>
+        <service>wicked</service>
+        <service>wickedd-auto4</service>
+        <service>wickedd-dhcp4</service>
+        <service>wickedd-dhcp6</service>
+        <service>wickedd-nanny</service>
+        <service>YaST2-Firstboot</service>
+        <service>YaST2-Second-Stage</service>
+        <service>getty@tty1</service>
```

Comment:
Do we need such a long service list in enable section?

Reply:
Yes, it should be enabled by default installation. All autoyast settings here come from standard document and facts. So I do not recommend to change them again unless you have very strong reason or actual experience.